docs/source/contributing.rst

```diff
@@ -3,79 +3,79 @@
 ============
 Contributing
 ============
-We absolutely welcome any code contributions and we hope that this
-guide will facilitate an understanding of the DPF-Core code
-repository. It is important to note that while the DPF-Core software
-package is maintained by Ansys and any submissions will be reviewed
-thoroughly before merging, we still seek to foster a community that
-can support user questions and develop new features to make this
-software a useful tool for all users.  As such, we welcome and
-encourage any questions or submissions to this repository.
+We welcome any code contributions and hope that this
+guide will facilitate an understanding of the PyDPF-Core code
+repository. While this library is maintained by Ansys and
+submissions will be reviewed thoroughly before merging, we
+seek to foster a community that can support user questions
+and develop new features to make this libray a useful tool
+for all users. As such, we welcome and encourage any
+questions or submissions to this repository.
 
 
 Cloning the Source Repository
 -----------------------------
 
-You can clone the source repository from `DPF-Core
-GitHub <https://https://github.com/pyansys/DPF-Core>`_
+You can clone the ``pydpf-core`` repository from
+`GitHub <https://github.com/pyansys/pydpf-core>`_
 and install the latest version in development mode by running:
 
 .. code::
 
-    git clone https://github.com/pyansys/DPF-Core DPF-Core
-    cd DPF-Core
+    git clone https://github.com/pyansys/pydpf-core PyDPF-Core
```

Comment:
Note sure if changing the name was something I should do--but I am trying to be consistent.

Reply:
This is correct.  Thanks for fixing this.